### DISCUSSION TITLE
Love these tabs, but I sadly have to uninstall the plugin

### DISCUSSION THREAD

**Discussion by fonsuak** [5axes/TabPlus#11] [General]
I really loved it for the past months but I lost count of the amount of prints ruined by supports melted to the walls since this plugins' tab distance setting modifies the Support X/Y distance.

Using the Support X/Y distance is choosing between having useless tabs or supports glued to the model walls. 
I also missed the setting change quite a few times, saved profile changes, and the next print that used support was ruined by having them glued to the walls.

I've read it in the disclaimer, and I take for granted there is no workaround for this kind of plugin atm but I'd love if these tabs would use the brim distance setting instead of the support distance setting.

I'm unisntalling it for now because I don't want to keep losing time ruining supported prints, but I really loved your idea @5axes !

**Comment by 5axes** [comment]
Try the spoon anti warping plugin in this case

**Reply by fonsuak** [reply]
Alright, will try those. Thank you!
